Client.js

```diff
@@ -11,19 +11,48 @@ function Client(accessToken) {
   });
 
   this.calendarResource = new CalendarResource(this);
+  this.buildingResource = new BuildingResource(this);
 }
 
 function CalendarResource(client) {
   this.client = client;
   this.resourcePath = '/directory/v1/customer/my_customer/resources/calendars'
 }
 
+function BuildingResource(client) {
+  this.client = client
+  this.resourcePath = '/directory/v1/customer/my_customer/resources/buildings'
+}
+
+function list(qs, cb) {
+  if (typeof qs === 'function') {
+    cb = qs
+    qs = {}
+  }
+  return this.client.request({method: 'GET', uri: this.resourcePath, qs: qs}, cb)
+};
+
+function get(resourceId, qs, cb) {
+  if (!resourceId) {
+    throw new Error('[google-apps-admin-sdk] resourceId must be provided when trying to GET a single resource')
+  }
+  if (typeof qs === 'function') {
+    cb = qs
+    qs = {}
+  }
```

Comment:
Though I understand this would be more consistent, I thought we are no longer doing this pattern anymore 😛
https://github.com/lever/guides/blob/master/coffeescript.md#function-smells

Reply:
Definitely a matter of opinion when talking about a fresh code base, but since I don't want to major version or break consistency with the list() api, I think this is okay.